src/lang/Messages.properties

```diff
@@ -1629,6 +1629,8 @@ report.unknown.error=Failed to write to file {0}, see log for detail
 report.write.overwrite.dialog.title=Existing File
 report.write.overwrite.dialog.message= The selected file already exists. Do you want to replace it?
 report.write.permission.dialog.title=Permissions Failure
+report.write.diskspace.warning.dialog.message=Warning: There may not be enough disk space to continue operation.
```

Comment:
- "the operation"?
 - It should probably say how much space may not be enough (i.e. the 5MB).
 - The "Warning: " could be dropped if it's shown a warning dialogue (currently it's showing an information dialogue).